src/raster/buffer.rs

```diff
@@ -28,14 +31,17 @@ impl<T: GdalType> Buffer<T> {
         Buffer { shape, data }
     }
 
-    /// Destructures `self` into constituent parts.
+    /// De-structures `self` into constituent parts.
```

Comment:
I see where you're coming from, but I really hate this.

Reply:
WDYM? The respelling of the word, or the method all together? 

If the former, I'll change it
If the latter, I thought we needed something to pull it apart. Other options?

Reply:
The spelling with a hyphen.